runelite-client/src/main/java/net/runelite/client/plugins/dpscounter/DpsCounterPlugin.java

```diff
@@ -168,6 +174,12 @@ protected void shutDown()
 		members.clear();
 	}
 
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		sort();
```

Comment:
this should be checking at least the config group being changed, maybe also the key.